src/modules/uuv_att_control/uuv_att_control.hpp

```diff
@@ -122,6 +122,7 @@ class UUVAttitudeControl: public ModuleBase<UUVAttitudeControl>, public ModulePa
 	uORB::Subscription	_parameter_update_sub{ORB_ID(parameter_update)};
 
 	int	_vehicle_attitude_sp_sub{-1};	/**< vehicle attitude setpoint */
+	int _vehicle_rates_setpoint_sub{-1}; /**< vehicle bodyrates setpoint subscriber */
```

Comment:
Suggestion only, consider moving these to the newer uORB api.

```suggestion
	uORB::Subscription _vehicle_rates_setpoint_sub{ORB_ID(vehicle_rates_setpoint)}; /**< vehicle bodyrates setpoint subscriber */
```

Then you can skip the orb_subscribe(), orb_unsubscribe(), orb_check(), orb_copy() and simply do `_vehicle_rates_setpoint_sub.update(&_vehicle_rates_sp)` in place.

Reply:
Switched it to the new API in 7f2762746e5eb64800a10a78e65772fa1d2f71ec. Might be a good idea to change the other subscriptions as well.

Reply:
Yes I'd recommend updating when it's convenient. Subscriptions via the new API are faster and cheaper.